src/components/Payment/Payment.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from "react";
+import { GoogleLogin } from "react-google-login";
+import { Button } from "react-bootstrap";
+import { websocketHelper } from "../../api/WebsocketHelper";
+import { RequestType } from "../../api/ApiTypes.d";
+import { loadStripe } from "@stripe/stripe-js";
+import api from "../../api/ApiHelper";
+
+// After google sign in, ui does not refresh
+
+function Payment() {
+  let [premiumExpiration, setPremiumExpiration] = useState(new Date());
+
+  let [googleId, setGoogleId] = useState(localStorage.getItem('googleId'));
+
+  useEffect(() => {
+    if (googleId) {
+      api.hasPremium(googleId).then((expiration: Date) => {
+        setPremiumExpiration(expiration);
+      });
+    }
+  });
+
+
+  const stripePromise = loadStripe(
+    "pk_test_51I6N5ZIIRKr1p7dQOGhRRigwIMqgZ3XnoBdbfezFNFgLiR9iaW2YzkRP9kAADCzxSOnqLeqKDVxglDh5uxvY28Dn00vAZR7wQ9"
+  );
+
+  const onLoginSucces = (response: any) => {
+    localStorage.setItem("googleId", response.googleId);
+    setGoogleId(response.googleId);
+  };
+
+  const onPay = async () => {
+    websocketHelper.sendRequest({
```

Comment:
the wesocket.sendRequest should be in the API-Helper. The Method call should be like this:
api.getPaymentSession(googleId).then(id => {})